man/org.freedesktop.systemd1.xml

```diff
@@ -2650,6 +2654,15 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       <para><varname>Transient</varname> contains a boolean that indicates whether the unit was created as a
       transient unit (i.e. via <function>StartTransientUnit()</function> on the manager object).</para>
 
+      <para><varname>TransientOwners</varname> contains a list of owners of a transient unit, i.e. retain
+      privilege to enqueue jobs on the unit without reauthentication. Specifically, the PID and PIDFD inode number
+      are reported, as well as a flags field defined as follows:</para>
```

Comment:
"i.e." is short for "id est", "that is". Something is wrong with that sentence.
The number in "inode number" is wrong too.

Reply:
> "i.e." is short for "id est", "that is". Something is wrong with that sentence.

Hmm, I'm trying to clarify what "owners" are here. Any suggestions on improvement?

>  The number in "inode number" is wrong too.

the "PID" and "PIDFD inode number" are reported 🤔

Reply:
I'm also trying to make out the the meaning of an "owner" from this description.
Is it about privileges over control or about pinning the unit active?

And why (and for whom) it needs to be exposed as DBus property?